.github/scripts/check-cli-binaries.sh

```diff
@@ -0,0 +1,77 @@
+echo "-------------- Checking CLI/Engines QE Binary --------------"
+DIR=$1
+PROJECT=$2
+
+# These are skipping because they have different project structures
```

Comment:
```suggestion
# These are skipping because they have different project structures
# TODO Adapt tests so they also work here, or adapt project to fit into the mold
```